src/components/validation/resources/http.rs

```diff
@@ -73,6 +74,7 @@ impl HttpResourceConfig {
         }
     }
 
+    #[allow(clippy::too_many_arguments)]
```

Comment:
😢

I don't think I'd demand this be converted to a struct, but I do wonder if it would make sense to do so, and make all of the `spawn*output*` functions as methods on that struct.